docs/users/coordinator.md

```diff
@@ -0,0 +1,277 @@
+# Coordinator
+This tutorial focuses on how to launch a Hermez Coordinator node on localhost using the available tools. It includes two parts:
+1. [Launching a Coordinator Node](#coordinator-node)
+2. [Launching a Proof Server](#proof-server)
+
+## Coordinator Node
+### Dependencies
+- golang 1.14+
+- [aws cli 2](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html)
+- packr utility to bundle the database migrations. Make sure your `$PATH` contains `$GOPATH/bin`, otherwise the packr utility will not be found.
+```shell
+cd /tmp && go get -u github.com/gobuffalo/packr/v2/packr2 && cd -
+```
+- docker and docker-compose without sudo permission
+
+### Setup
+1. Clone [hermez-node](https://github.com/hermeznetwork/hermez-node.git) repo
+```shell
+git clone https://github.com/hermeznetwork/hermez-node.git
+```
+2. Build `hermez-node` executable
+```shell
+cd hermez-node/db && packr2 && cd ../cli/node
+go build .
+cd ../../db && packr2 clean && cd -
+```
+The executable can be found in cli/node/node
+
+
+3. Deploy postgresql database and geth node. For this step we provide a docker-compose file example. Copy contents to file named `docker-compose.sandbox.yaml`
```

Comment:
I'd add an alternative to whom does not want to use docker or has AWS dependencies to install GETH using one of these methods: 

- Pre-built binaries for all platforms on our downloads page (https://geth.ethereum.org/downloads/).
- Ubuntu packages in our Launchpad PPA repository (https://launchpad.net/~ethereum/+archive/ubuntu/ethereum).
OSX packages in our Homebrew Tap repository (https://github.com/ethereum/homebrew-ethereum).

And sync this node with Goerli testnet where all Hermez's smart contract are deployed. To do this the only change at GETH's initialization is to change the parameter --dev --goerli

Reply:
If you don't mind, I wil not add the non-docker installation. If you know what you're doing, you can launch your own ethereum node. I am not getting into this because then i need to go through the contract deployment and i much rather save this for the second phase.
In this second phase, i will describe how to launch a node in testnet (goerli, rinkeby...), using predeployed contracts, and i will talk about this. also i will explain how to launch several coordinators to run the bidding

Reply:
As per our conversation is worthy to add these lines to help and to be clear to Exchange infra experts they can use directly GETH if they want to instead to use Docker.